pallets/avn/src/lib.rs

```diff
@@ -181,20 +175,14 @@ pub mod pallet {
 
     #[pallet::storage]
     #[pallet::getter(fn get_primary_collator)]
-    pub type PrimaryCollator<T: Config> = StorageValue<_, PrimaryCollatorData, ValueQuery>;
+    pub type PrimaryCollator<T: Config> = StorageValue<_, u8, ValueQuery>;
```

Comment:
Can we rename this to make it more explicit? Something like `primaryCollatorIndexForEth`? I also think we should use a large type for the value. u8 give 255 only and its possible we will end up sending more than that. Perhaps a `u32`?